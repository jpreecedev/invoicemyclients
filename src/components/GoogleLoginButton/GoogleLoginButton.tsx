import React from 'react'
import clsx from 'clsx'
import { Button, Icon } from 'semantic-ui-react'

import styles from './styles.module.css'

const GoogleLoginButton = () => (
  <Button fluid size="big" className={clsx(styles.space, 'google plus')}>
    <Icon name="google" />
    Log In Using Google
  </Button>
)

export { GoogleLoginButton }
