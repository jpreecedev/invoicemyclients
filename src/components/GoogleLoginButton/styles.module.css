.space {
  margin-bottom: 16px !important;
}
