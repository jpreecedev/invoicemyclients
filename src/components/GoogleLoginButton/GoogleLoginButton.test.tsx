import React from 'react'
import { render } from '@testing-library/react'
import { GoogleLoginButton } from './GoogleLoginButton'

describe('<GoogleLoginButton /> tests', () => {
  it('should match snapshot', () => {
    expect(render(<GoogleLoginButton />).container).toMatchSnapshot()
  })
})
