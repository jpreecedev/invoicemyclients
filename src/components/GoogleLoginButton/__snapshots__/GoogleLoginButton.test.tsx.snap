// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GoogleLoginButton /> tests should match snapshot 1`] = `
<div>
  <button
    class="ui big fluid button space google plus"
  >
    <i
      aria-hidden="true"
      class="google icon"
    />
    Log In Using Google
  </button>
</div>
`;
